SourceryRuntime/Sources/Definable.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Describes that the object is defined in a context of some `Type`
+public protocol Defined: class {
```

Comment:
Maybe we can really get rid of current `Definition`? I don't see the benefit of having a protocol with completely default implementation that is used for one type.
Check 5240cf6